project/MimaBuild.scala

```diff
@@ -88,7 +88,7 @@ object MimaBuild {
 
   def mimaSettings(sparkHome: File, projectRef: ProjectRef) = {
     val organization = "org.apache.spark"
-    val previousSparkVersion = "2.0.0"
+    val previousSparkVersion = "2.2.0"
```

Comment:
does this matter if this is 2.2.1 (latest release)?

Reply:
I checked the change history, we always use x.y.0 release. cc @JoshRosen

Reply:
Yeah, 2.2.0 is right.